src/types/MainContentType.ts

```diff
@@ -0,0 +1,7 @@
+export enum MainContentType {
+  NoSelectedUser,
+  Loader,
+  PostsLoadingError,
+  NoPostsYet,
+  PostsList,
+}
```

Comment:
👎🏻 
1. it is not a type
2. it is not a good practice to handle more then one case with enum. create a separate enums for error messages, handle error case with state, etc.